Long camera transitions to use flyTo when in following mode (#5871)
mapbox/mapbox-navigation-android: CHANGELOG.md

## Unreleased
#### Features
#### Bug fixes and improvements

> - :warning: Fixed long camera transitions to use the default `flyTo` animation supported by `mapboxMap.flyTo` when transitioning in following mode. [#5871](https://github.com/mapbox/mapbox-navigation-android/pull/5871)

Reviewer: ```suggestion
- :warning: Changed  default `NavigationCamera` transitions to use the `flyTo` animation when transitioning to `NavigationCameraState#FOLLOWING` mode. [#5871](https://github.com/mapbox/mapbox-navigation-android/pull/5871)
```
